frontend/package.json

```diff
@@ -32,6 +32,8 @@
     "jsdom": "^20.0.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.5.1",
+    "sass": "^1.54.3",
+    "sass-loader": "^13.0.2",
```

Comment:
Na Slacku sme sa dohodli, že budeme používať scss a sass zahodíme. Prosím upraviť :).

Reply:
To nejde. Scss  (Sassy CSS) je Sass. Jenom jinej flavor.

Reply:
Učím sa každý deň :). Vďaka za info.